Gopkg.lock

```diff
@@ -22,6 +22,14 @@
   revision = "ab2d9a74b97eda058004c8deef80ac624432f408"
   version = "v1.0.0"
 
+[[projects]]
+  digest = "1:141cc9fc6279592458b304038bd16a05ef477d125c6dad281216345a11746fd7"
+  name = "github.com/gofrs/uuid"
```

Comment:
Hi @ataleksandrov! Sorry, we just noticed you added a new UUID generation dependency. We are already using https://github.com/pborman/uuid in this project. Do you have a strong objection to using the package we already have? If not, could you update your code to use the pborman dependency? If you do want to use this new package, could you swap out the pborman package where we use it for this one to minimise dependencies and make sure to update the Gopkg.toml?